src/applications/personalization/profile/mocks/server.js

```diff
@@ -40,7 +42,26 @@ const responses = {
   'GET /v0/user': user.handleUserRequest,
   'GET /v0/profile/status': status.success,
   'OPTIONS /v0/maintenance_windows': 'OK',
-  'GET /v0/maintenance_windows': { data: [] },
+  'GET /v0/maintenance_windows': (_req, res) => {
+    // three different scenarios for testing downtime banner
+    // all service names/keys are available in src/platform/monitoring/DowntimeNotification/config/externalService.js
+    // but couldn't be directly imported due to export default vs module.exports
+
+    // return res.json(
+    //   maintenanceWindows.createDonwtimeApproachingNotification([
```

Comment:
I think there is a typo here. Should it be `createDowntimeApproachingNotification`?

Reply:
yikes lol. Can't believe I borked these. Will fix, Thanks

Reply:
Happens to the best of us, at least they were consistent lol